ocaml/xapi/xapi_cluster_helpers.ml

```diff
@@ -109,6 +109,11 @@ let cluster_health_enabled ~__context =
   let restrictions = Db.Pool.get_restrictions ~__context ~self:pool in
   List.assoc_opt "restrict_cluster_health" restrictions = Some "false"
 
+let corosync3_enabled ~__context =
```

Comment:
There is a `restrict_corosync` already. Are we going to separate features for clustering?

Reply:
Yes see comments below